rasa_core/evaluate.py

```diff
@@ -81,6 +81,16 @@ def create_argument_parser():
             help="If a prediction error is encountered, an exception "
                  "is thrown. This can be used to validate stories during "
                  "tests, e.g. on travis.")
+    parser.add_argument(
+            '--mode',
+            type=str,
+            default='default',
```

Comment:
this shouldn't be a string but rather a "choice":
`'--mode', choices=('default', 'compare'), help= ...`